dev/tab1tha/k_nn.py

```diff
@@ -0,0 +1,53 @@
+from sklearn.neighbors import KNeighborsClassifier
+import matplotlib.pyplot as plt
+import numpy as np
+
+def k_nearest(data_train, target_train, data_test, target_test):
+    #training model using the k nearest neighbor algorithm
```

Comment:
Please replace multi-line comments with docstrings as per relevant [PEP conventions](https://www.python.org/dev/peps/pep-0257/)

Reply:
In this case, due to the fact that turning the #comment into a docstring might make it to be interpreted as part of the  function's docstring, can I move the second comment line that begins with "note that ..." from line 7 to line 11, so that there would not be concurrent #comments as observed now in lines 6 and 7?

Reply:
I have merged both comments into the main docstring as you recommended.